.eslintrc.js

```diff
@@ -77,8 +77,7 @@ module.exports = {
         'import/prefer-default-export': 0,
         'import/extensions': ['warn', 'never', { jsx: 'always', json: 'always' }],
         'no-sequences': ['warn'],
-        'import/no-unresolved': [2, { ignore: ['@deriv/components', '@deriv/shared'] }],
-
+        'import/no-unresolved': [2, { ignore: ['@deriv/components', '@deriv/shared', '@deriv/components/cashier'] }],
```

Comment:
@ashraf-deriv why are adding @deriv/components/cashier?

Reply:
Were added for testing the Eslint error, No need of that anymore Reverting.